loading_NYC_taxi_data.ipynb

```diff
@@ -0,0 +1,854 @@
+{
```

Comment:
<p>Have you ever used this before? https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.apply.html</p><p> </p><p>You can DRY up your code by writing a function for date cleaning and applying to the date columns for all of your dataframes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/harrislam1/covid19_NYC_taxi_trips/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>